docs/trex_publish.md

```diff
@@ -62,12 +62,12 @@ In addition to the requirements for all extensions to ensure security and usabil
  * Declaring data access requirements
  * Providing a URL that customers can use to get support for your the extension
  
- Dashboard extensions that appear in the gallery must also:
+ Dashboard extensions that appear in the Exchange must also:
 
 * Follow the Design Guidelines for Dashboard Extensions (user interaction and style guidelines for user interface elements).
 * Ensure that the information in the extension manifest file (`.trex`) matches the content that you will publish in the Extension Gallery. For example, the `name`, `description` fields are used to populate the name and description fields in the Extension Gallery.  The icon you use in the manifest should also be the icon that is used in the gallery. You need to provide a 280x280 pixel `.png` version of the icon. 
```

Comment:
Couple more instances of Extension Gallery here if you want to change them

Reply:
done